vividus-extension-selenium/src/test/java/org/vividus/bdd/steps/ui/model/StringSortingOrderTests.java

```diff
@@ -27,18 +28,28 @@
 
 class StringSortingOrderTests
 {
+    private static final String KEY_1 = "Cry Over Spilt Milk";
+    private static final String KEY_2 = "an Arm and a Leg";
+    private static final String KEY_3 = "A Piece of Cake";
+    private static final String KEY_4 = "Beating Around the Bush";
+
     private static Stream<Arguments> dataProvider()
     {
         return Stream.of(
-            Arguments.of(StringSortingOrder.ASCENDING, Comparator.naturalOrder()),
-            Arguments.of(StringSortingOrder.DESCENDING, Comparator.reverseOrder())
+            Arguments.of(StringSortingOrder.ASCENDING, List.of(KEY_3, KEY_4, KEY_1, KEY_2)),
+            Arguments.of(StringSortingOrder.DESCENDING, List.of(KEY_2, KEY_1, KEY_4, KEY_3)),
+            Arguments.of(StringSortingOrder.ASCENDING_IGNORE_CASE, List.of(KEY_3, KEY_2, KEY_4, KEY_1)),
+            Arguments.of(StringSortingOrder.DESCENDING_IGNORE_CASE, List.of(KEY_1, KEY_4, KEY_2, KEY_3))
         );
     }
 
     @ParameterizedTest
     @MethodSource("dataProvider")
-    void shouldReturnSortingComparator(StringSortingOrder sortingOrder, Comparator<? extends Comparable> comparator)
+    void shouldReturnSortingComparator(StringSortingOrder sortingOrder, List<String> expected)
     {
-        assertEquals(sortingOrder.getSortingType(), comparator);
+        List<String> sorterd = List.of(KEY_1, KEY_2, KEY_3, KEY_4).stream()
+                                                                  .sorted(sortingOrder.getSortingType())
+                                                                  .collect(Collectors.toList());
+        assertEquals(expected, sorterd);
```

Comment:
List.of(KEY_1, KEY_2, KEY_3, KEY_4).sort(sortingOrder.getSortingType()) ?

Reply:
it will fail because List.of is immutable

Reply:
Arrays.asList(...) then? :)

Reply:
`sorterd` -> `sorted`

Reply:
sort returns void, so imho sort via stream looks good

Reply:
JMH time :)

Reply:
https://stackoverflow.com/questions/49798129/what-is-more-efficient-sorted-stream-or-sorting-a-list
![image](https://user-images.githubusercontent.com/17816729/102337812-04a46080-3fa4-11eb-89c7-482fd1af10ad.png)

Reply:
okay, but
- this is just a unit test
- lets stop using streams where we don't need parallel computations?